vizro-core/tests/unit/vizro/conftest.py

```diff
@@ -25,20 +23,6 @@ def standard_px_chart(gapminder):
     )
 
 
-@pytest.fixture
```

Comment:
My only concern is, will these very standard fixtures be "found"?

Reply:
Currently, there are not that many fixtures and we managed to re-use many after reminding people during PRs 😄 

I think we just need to double-check every now and then and consolidate